docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -30,14 +30,137 @@ Understanding GBD's Disease and Health Categories
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
+through the disability weight. More information on this can be found in `this section of the cause model page <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#disability-weights>`_. A health state for a sequelae and its disability weight can be 
+found in the GBD methods appendix. 
+
+Using our above sequelae (acute myocardial infarction and angina) as an example - 
+the amount of health lost while experiencing an acute 
+myocardial infarction is quite high! You're likely hospitalized and in a lot of pain. 
+However, the health lost from angina might be less significant. You're likely at home and 
+might only experience discomfort when under stress. This affects your disability weight value. 
+
+Each sequela has exactly one health state associated with it. If you are experiencing multiple 
+symptoms, such as "headache and chest pain", your health state contains all of these losses to 
+health in one "health state". 
+
+One important note is that health states can be shared across sequelae - and even shared 
+between sequelae of different causes. For example, severe heart failure due to IHD and 
+severe heart failure due to HHD have the same amount of health loss associated with them. 
+Severe heart failure feels the same to a patient no matter the cause. Therefore, we would 
+say that the health state and therefore the disability weight is the same for both sequelae. 
+
+Impairments
++++++++++++
+
+Sometimes, GBD has a lot of information about a health state - such as severe heart failure in the 
+example above - that the modelers want to utilize. However, in the process of normal GBD 
+modeling, this wouldn't be possible. Usually, the cause level information is calculated 
+first and then divded into sequelae. 
+
+Therefore, to use this valuable information about a health state, GBD created a new 
+category called impairments. An impairment is medically the same as a sequela, but 
+there is more information about it. To include this, GBD modelers first find the total 
+amount of the impairment - heart failure - and then divide it between causes. This 
+creates the sequelae of "heart failure due to IHD" and "heart failure 
+due to HHD" for example. These are included in blue in the diagram above. 
+
+These sequelae therfore have the same health state. Therefore, an 
+impairment is a grouping of the related health states (mild/moderate/severe heart failure for 
+example) that show up in 2 or more causes. There are many examples that fit this 
+definition but are not GBD impairments, since the main criteria is the amount of 
+information about the health state. 
```

Comment:
```suggestion
definition but are not GBD impairments, since the main criteria for being a GBD impairment
is the amount of data available about the health state(s). 
```